epplib/commands/info.py

```diff
@@ -45,8 +45,10 @@ def _get_info_payload(self, namespace: str, schema_location: str,
         SubElement(domain_info, QName(namespace, tag)).text = item
 
         if auth_info is not None:
-            SubElement(domain_info, QName(namespace, "authInfo")).text = auth_info
-
+            if isinstance(auth_info, str):
```

Comment:
That's a good point. I don't think it is worthwhile to generalize, since auth info for domains and auth info for contacts are only coincidentally shaped the same.